### ISSUE TITLE
Showing a Users Posting does not provide links to users profile

### ISSUE THREAD

**Issue by xundeenergie** [friendica/friendica#12473]
### Is the feature request related to a problem? Please describe.

When i follow a Link in a notification, i see the users Avatar and the posting from the notification
![grafik](https://user-images.githubusercontent.com/5894829/208353950-62577a7e-fa75-43e1-800f-b1215042275c.png)
I can follow/unfollow this user.

But if i want to see other postings from this user, i have to search for the handle to get the contact found, clicking on it shows this

![grafik](https://user-images.githubusercontent.com/5894829/208354073-f8aa5010-fdf7-4bec-b494-4f215a856df9.png)

This is not very intuitive. 

### Describe the feature you'd like

If i click on a notification, i want to see the users profile like in contact-view. And the postings/status of this user is sorted in a way, the posting from the notification is the first, then the others come in chronological order.

**Comment by tobiasd** [comment]
> Showing a Users Posting does not provide links to users profile

You should be able to click on the profile picture of the original poster to get to their profile.

**Comment by MrPetovan** [comment]
This feature is already available by enabling “Stay Local” in Settings -> Display -> Content/Layout.

Most external links to remote profiles will be changed to links to their conversations.

**Comment by xundeenergie** [comment]
yes... but i have disabled "stay local"... so adding an extra button would work in both environments.

**Comment by MrPetovan** [comment]
The setting exists so that we don't have to add an extra button in an already busy interface.

**Comment by AlfredSK** [comment]
I just tried it. After following the notification i am able to open the hover menu and click on the network (cloud) icon to get that user's posts. No need to invoke a search.
Or do I miss something?

**Comment by MrPetovan** [comment]
Is it possible @xundeenergie isn't aware of the contact card menu on hovering display names or clicking the profile picture?
